docs/chaos/outcome.md

```diff
@@ -161,17 +161,151 @@ new ResiliencePipelineBuilder<HttpResponseMessage>()
+
+<!-- snippet: chaos-outcome-pattern-generator-inject-fault -->
+```cs
+var randomThreshold = Random.Shared.Next(350);
+var pipeline = new ResiliencePipelineBuilder<HttpResponseMessage>()
+    .AddChaosFault(new ChaosFaultStrategyOptions
+    {
+        InjectionRate = 0.1, // Different injection rate for faults
+        EnabledGenerator = static args => ValueTask.FromResult(ShouldEnableFaults(args.Context)), // Different settings might apply to inject faults
+        FaultGenerator = args =>
+        {
+            Exception? exception = randomThreshold switch
+            {
+                >= 250 and < 350 => new HttpRequestException("Chaos request exception."),
                 _ => null
             };
 
+            return new ValueTask<Exception?>(exception);
+        },
+        OnFaultInjected = static args =>
+        {
+            Console.WriteLine($"OnFaultInjected, Exception: {args.Fault.Message}, Operation: {args.Context.OperationKey}.");
+            return default;
+        }
+    })
+    .AddChaosOutcome(new ChaosOutcomeStrategyOptions<HttpResponseMessage>
+    {
+        InjectionRate = 0.5, // Different injection rate for outcomes
+        EnabledGenerator = static args => ValueTask.FromResult(ShouldEnableOutcome(args.Context)), // Different settings might apply to inject outcomes
+        OutcomeGenerator = args =>
+        {
+            Outcome<HttpResponseMessage>? outcome = randomThreshold switch
+            {
+                < 100 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.InternalServerError)),
+                < 150 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.TooManyRequests)),
+                < 250 => Outcome.FromResult(CreateResultFromContext(args.Context)),
+                _ => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.OK))
+            };
+
             return ValueTask.FromResult(outcome);
+        },
+        OnOutcomeInjected = static args =>
+        {
+            Console.WriteLine($"OnBehaviorInjected, Outcome: {args.Outcome.Result}, Operation: {args.Context.OperationKey}.");
+            return default;
         }
+    })
+    .Build();
+```
+<!-- endSnippet -->
+
+❌ DON'T
+
+Use outcome strategies to inject only faults, use the [`ChaosFaultStrategy`](fault.md) instead.
+
+<!-- snippet: chaos-outcome-anti-pattern-only-inject-fault -->
+```cs
+new ResiliencePipelineBuilder<HttpResponseMessage>()
+    .AddChaosOutcome(new ChaosOutcomeStrategyOptions<HttpResponseMessage>
+    {
+        OutcomeGenerator = new OutcomeGenerator<HttpResponseMessage>()
+            .AddException<HttpRequestException>(), // ⚠️ Avoid this ⚠️ 
```

Comment:
Please do **NOT** edit the snippets inside the markdown file.

It will be overwritten the very next time when someone runs the `dotnet mdsnippets` command.

I suggest to move the `// ⚠️ Avoid this ⚠️` comments into the source code as well.

Reply:
Nice! I didn't know